sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_shared/http_challenge_cache.py

```diff
@@ -3,13 +3,10 @@
 # Licensed under the MIT License.
 # ------------------------------------
 import threading
-from typing import TYPE_CHECKING
+from typing import Dict, Optional
 from urllib import parse
 
-if TYPE_CHECKING:
-    # pylint: disable=unused-import
-    from typing import Dict, Optional
-    from .http_challenge import HttpChallenge
+from .http_challenge import HttpChallenge
```

Comment:
This one is not part of the pylint rule, and may have perf impact. I'm not against it,  just want to be clear it's on purpose